### DISCUSSION TITLE
Can we have the Home Assistant Name back?

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#7443] [Other feature requests]
As a major indicator for which instance (of the 3) I am currently interfacing, I have always been able to discern these from the Homeassistant Name in the (each) View.

I've even been able to add the View name to that, with the amazing Custom Header, that gave us these internal variables.

Now 117 has landed we see the compact header, which takes out the name of the Instance (not even speaking of the custom View name)

I feel this is a major drawback in the user interface. All instances now show Homeassistant  (duh) in the left side panel, and none of the views shows the name (while we still can edit it....)
Only those of us with only 1 view (...) can see the name.

The name we all set as a first setting in the configuration.yaml 

So this is a request to give us the Name of the instance back. 

If this wouldn't be an option for the right side of the view, please consider placing the name in the left side panel, where we now see the indiscriminate Homeassistant.

116
with favicon and Custom Header:
<img width="624" alt="Schermafbeelding 2020-10-22 om 12 03 47" src="https://user-images.githubusercontent.com/33354141/96880309-52f22300-147d-11eb-8984-c2eca49b9bc3.png">

116 only favicon

<img width="519" alt="Schermafbeelding 2020-10-22 om 12 03 40" src="https://user-images.githubusercontent.com/33354141/96880374-643b2f80-147d-11eb-85cd-e27ca4356c4a.png">

and the latest 117.0b1
without favicon, <strike>but that seems not to work</strike> was uninstalled somehow
<img width="612" alt="Schermafbeelding 2020-10-22 om 12 03 34" src="https://user-images.githubusercontent.com/33354141/96880503-846aee80-147d-11eb-90f8-89fa8a797045.png">

thanks for considering users that have the need for more than 1 instance

**Comment by balloob** [comment]
Did you consider to configure a different theme?

**Reply by Mariusthvdb** [reply]
> Did you consider to configure a different theme?

Well yeah, I did and tried. Though, since already using various themes for home settings (day, night, alarm) it would seem less obvious adding this too.

Also, Name being the major indicator for the instance, it doesn’t seem obvious to have to set it(represent it by a color) in a theme, nor that it isn’t available when more than 1 view is used .

Really hoping Ryan’s Pr implementing the suggestion I did gets merged.

Making the current HomeAssistant actually meaningful in the left side panel.
Thanks!

**Reply by maykar** [reply]
Continuing the conversation from the PR that is no longer related to this issue:

This is the place to discuss this or on the forum's feature request section.

I can understand balloob's point on this as it is a bit of an edge case to have more than one instance of Home Assistant and the solution proposed in the previous PR feature would force removal of all branding if you name your instance, rather than making it optional.

**Comment by Mariusthvdb** [comment]
Thanks Ryan,

real sorry for the simple and '[undiscussable](https://github.com/home-assistant/frontend/pull/7470#issuecomment-716043377)' position that has been taken now. Especially after Paulus said that 

> We need to make sure that feature requests fit within our vision for what the frontend is, and what it is not.

Believed that to be in invitation to do just that, discuss.

Can't understand why we would need the really pointless name Homeassistant to be so prominently in view on all views in the interface. I mean, it's not that we might think we're using Smarthings is it. And it doesn't add anything useful, might as well take it out completely then. Why have a name: field in the config, if we can't see it in the Frontend anymore. 

Where do we show 'Our Awesome Home' now ..

thanks for the custom project. will do that. Though please know Thomas Lovens Favicon does that too, and has some more tricks, so the browser tab name changes too with a favicon

Guess we can close this FR if the pov is final.

though, as we see requests for the support of multiple instances on the HA App increase, this might not be such an edge case as you say it is.

**Reply by maykar** [reply]
Thought from your previous comments that you no longer had a solution to this, but if Favicon does the trick I'll kill the other custom project as it seems like it checks all the boxes you need and more.

**Comment by Mariusthvdb** [comment]
closing because realized by different means
